src/encoded/static/components/static-pages/placeholders/JointAnalysisMatrix.js

```diff
@@ -300,12 +300,16 @@ class VisualBody extends React.PureComponent {
     }
 
     static blockRenderedContents(data, title, groupingPropertyTitle, blockProps){
+        var count = 0;
         if (Array.isArray(data)) {
-            return data.length;
+            count = data.length;
         } else if (data) {
-            return 1;
+            count = 1;
         }
-        return 0; // Shouldnt happen?
+        if (count > 100){
+            return <span className="small">{ count }</span>;
```

Comment:
Smaller font-size.
We have a few different non-consistent classNames for text sizes that should consider making consistent (maybe from Bootstrap?), including `small` (0.925rem?), `text-small` (0.9rem?), `smaller` & `text-smaller`.

Reply:
Replaced with inline styling (0.95rem) which looks a bit better.